Failed to launch

```
java -jar shadow-0.0.1-SNAPSHOT-all.jar
Graphics Device initialization failed for :  es2, sw
Error initializing QuantumRenderer: no suitable pipeline found
java.lang.RuntimeException: java.lang.RuntimeException: Error initializing QuantumRenderer: no suitable pipeline found
        at com.sun.javafx.tk.quantum.QuantumRenderer.getInstance(QuantumRenderer.java:280)
        at com.sun.javafx.tk.quantum.QuantumToolkit.init(QuantumToolkit.java:222)
        at com.sun.javafx.tk.Toolkit.getToolkit(Toolkit.java:260)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:267)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:158)
        at com.sun.javafx.application.LauncherImpl.startToolkit(LauncherImpl.java:658)
        at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:678)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:195)
        at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: java.lang.RuntimeException: Error initializing QuantumRenderer: no suitable pipeline found
        at com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.init(QuantumRenderer.java:94)
        at com.sun.javafx.tk.quantum.QuantumRenderer$PipelineRunnable.run(QuantumRenderer.java:124)
        ... 1 more
Exception in thread "main" java.lang.RuntimeException: No toolkit found
        at com.sun.javafx.tk.Toolkit.getToolkit(Toolkit.java:272)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:267)
        at com.sun.javafx.application.PlatformImpl.startup(PlatformImpl.java:158)
        at com.sun.javafx.application.LauncherImpl.startToolkit(LauncherImpl.java:658)
        at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:678)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:195)
        at java.base/java.lang.Thread.run(Thread.java:832)
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -13,6 +13,8 @@ group = "com.pokosho.midi2musicxml"
 version = "0.0.1-SNAPSHOT"
 java.sourceCompatibility = JavaVersion.VERSION_1_8
 
+val javafxVersion = "11.0.2"
+
 repositories {
   mavenCentral()
 }
@@ -26,6 +28,9 @@ dependencies {
   implementation("com.atilika.kuromoji:kuromoji-ipadic:0.9.0")
   implementation("org.openjfx:javafx-controls:14")
   implementation("no.tornado:tornadofx:1.7.17")
+  compileOnly("org.openjfx:javafx-graphics:$javafxVersion.version:win")
+  compileOnly("org.openjfx:javafx-graphics:$javafxVersion:linux")
+  compileOnly("org.openjfx:javafx-graphics:$javafxVersion:mac")
 
   testImplementation("org.mockito:mockito-junit-jupiter:3.3.3")
   testImplementation("org.junit.jupiter:junit-jupiter-engine:5.6.2")
@@ -47,7 +52,7 @@ tasks.withType<KotlinCompile> {
 
 tasks {
   named<ShadowJar>("shadowJar") {
-    archiveBaseName.set("shadow")
+    archiveBaseName.set("midi2musicxml")
     mergeServiceFiles()
     manifest {
       attributes(mapOf("Main-Class" to "com.github.csolem.gradle.shadow.kotlin.example.App"))
@@ -62,7 +67,7 @@ tasks {
 }
 
 javafx {
-  version = "11.0.2"
+  version = javafxVersion
   modules = listOf("javafx.controls", "javafx.graphics", "javafx.fxml")
 }
 

